src/sites/image/xxx.fopkodiak.site.js

```diff
@@ -0,0 +1,22 @@
+_.register({
+  rule: {
+    host: /^xxx\.fopkodiak\.site$/,
+    path: /img-.*/,
+  },
+  async ready () {
+    if (document.referrer == document.location.href) {
+      let img = $.$('#container > a > img');
+      if (img === null) {
```

Comment:
This rule looks similar to `imgsee.me.js`, I think you can safely put this into `imgsee.me.js`.

Reply:
i think we can support the site, and later on in near future do some refactoring.